lego/apps/surveys/permissions.py

```diff
@@ -33,6 +40,8 @@ def has_permission(self, request, view):
             event=event.id, user=user.id, presence=constants.PRESENT
         ).exists()
 
+        if view.action in ['update', 'partial_update']:
+            return False
```

Comment:
This means admins can't edit Submissions, which seems like a fine personal integrity measure to me.